packages/common/core-utils/api-report/core-utils.api.md

```diff
@@ -138,7 +138,7 @@ export function setLongTimeout(timeoutFn: () => void, timeoutMs: number, setTime
 
 // @internal
 export class Timer implements ITimer {
-    constructor(defaultTimeout: number, defaultHandler: () => void, getCurrentTick?: () => number);
+    constructor(defaultTimeout: number, defaultHandler: () => void, exceptionHandler?: ((error: unknown) => void) | undefined);
```

Comment:
If we specified the type of error here as any, then we could keep the other error types preserved which you had to change below to unknown.

Reply:
I'm not sure what you mean. unknown is safer than any, as any could be implicitly casted by compiler to any type, so it's easy to introduce a bug without noticing it.

The only casts I see is in UT and they are unrelated to this handler - they are there because compiler does not understand that clock.tick() has side-effects, so it thinks that exceptionCounter is always one (after first assert).